ios/brave-ios/App/l10n/tools/download-translations-from-transifex.swift

```diff
@@ -7,6 +7,48 @@ guard let apiKey = ProcessInfo.processInfo.environment["PASSWORD"] else {
   exit(EXIT_FAILURE)
 }
 
+struct LanguageMapping {
+  /// The language code on the server.
+  let source: String
+  /// Sometimes we may want to have one language code to be used in many languages or dialects.
+  /// This array has a list of those additional languages.
+  /// For example German may have Swiss-German variant as their additional target.
+  /// This list is empty if there is no additional targets.
+  let additionalTargets: [String]
+
+  /// Returns base language code and all additional targets we want to support.
+  var targets: [String] {
+    [source] + additionalTargets
+  }
+
+  init(source: String, additionalTargets: [String] = []) {
+    self.source = source
+    self.additionalTargets = additionalTargets
+  }
+
+  static let allMappings: [LanguageMapping] = [
+    LanguageMapping(source: "fr"),
+    LanguageMapping(source: "pl"),
+    LanguageMapping(source: "ru"),
+    LanguageMapping(source: "de", additionalTargets: ["gsw"]),
```

Comment:
I added Swiss German here as we want to have it next time we pull. Lmk if it's good to keep it here or should I remove add add in a followup ticket